src/IdenityServer4InMem/Views/Account/LoggedOut.cshtml

```diff
@@ -3,6 +3,8 @@
 @{ 
     // set this so the layout rendering sees an anonymous user
     ViewData["signed-out"] = true;
+    Model.AutomaticRedirectAfterSignOut = true;
+    Model.PostLogoutRedirectUri = "https://localhost:5002";
```

Comment:
this property "AutomaticRedirectAfterSignOut" may be set  to true when we build up LoggedOutViewModel object

Reply:
I fixed everything I could, but can't get it working without this line.

Reply:
if you first of all set public static bool AutomaticRedirectAfterSignOut = true; in AcccountOptions.cs (IdSvr4 project) and add  // where to redirect to after logout
PostLogoutRedirectUris = { "https://localhost:5002/signout-callback-oidc" } on client in Config.cs (IdSvr4 project), then you don't need to have these lines here. Thanks